src/textual/widgets/_data_table.py

```diff
@@ -169,13 +132,23 @@ class DataTable(ScrollView, Generic[CellType], can_focus=True):
 
     COMPONENT_CLASSES: ClassVar[set[str]] = {
         "datatable--header",
+        "datatable--cursor-fixed",
+        "datatable--highlight-fixed",
```

Comment:
We'll probably need to change the names of these when we do "labelled" rows/columns - the fixed/header concepts are a little conflated at the moment.